forwards_paper.tex

```diff
@@ -374,17 +379,17 @@ \subsection*{Working with a tree sequence through tables}
     \item The set of intervals on which each individual is a child must be disjoint.
 \end{enumerate}
 A pair of node and edge tables that satisfy these two requirements
-is guarenteed to uniquely describe at each point on the genome
+is guaranteed to uniquely describe at each point on the genome
 a collection of directed, acyclic graphs -- in other words, a forest of trees.
 For some applications it is necessary to check that at every point
 there is only a \emph{single} tree.
 Checking this is more difficult, but is implemented in \msprime{}'s API.
-\jda{TODO add function name}
+% \jda{TODO add function name}  PLR: not sure this is necessary.
```

Comment:
good thought, but would make the paper read more like documentation for msprime, which we already run the risk of doing...

Reply:
agreed. I think this early suggestion of mine runs counter to how I feel now